### PR TITLE
Parsing the connection b/w ports

### PR DESCRIPTION
**Pull request by douglasgabriel**
__This PR is able to__
- Fill the switches physical ports `uid_ems`;
- Bind the ports from Physical Servers and Physical Switches;

__Informations__
A port is connected to another if its MAC address is described in the `peer_mac_address` of the other port.

__Depends on__
~https://github.com/ManageIQ/manageiq-schema/pull/208~ - Merged
~https://github.com/ManageIQ/manageiq/pull/17311~ Merged

### CODE REVIEW

**Comment by douglasgabriel** [comment]
@miq-bot assign @agrare

File: spec/manageiq-schema [hunk 1]
```diff
@@ -0,0 +1 @@
+/home/douglas/workspace/manageiq-schema
```
Comment (agrare) [inline spec/manageiq-schema]: Delete this

**Comment by agrare** [comment]
@rodneyhbrown7 assigning to you to review, looks fine from the MIQ side but need you to review the LXCA stuff

**Comment by agrare** [comment]
@douglasgabriel or if you can get a review from another member of your team I'll merge this given how long it has been sitting here

**Comment by douglasgabriel** [comment]
@caiocmpaes could you review that, please?

File: app/models/manageiq/providers/lenovo/physical_infra_manager/refresh_parser.rb [hunk 2]
```diff
@@ -139,6 +141,18 @@ def get_config_patterns
       config_patterns = @connection.discover_config_pattern
       config_patterns.map { |config_pattern| @parser.parse_config_pattern(config_pattern) }
     end
+
+    def bind_network_ports(inventory)
```
[the remark this answers is not in the document]
Comment (douglasgabriel) [reply app/models/manageiq/providers/lenovo/physical_infra_manager/refresh_parser.rb]: Good point, @caiocmpaes, it's a fair concern, but our methods in this class doesn't use this pattern and to keep the consistence I would say to keep this name. Also the method that is really doing the changes has already its name ending with `!`. What do you think? :thinking:

[the remark this answers is not in the document]
Comment (caiocmpaes) [reply app/models/manageiq/providers/lenovo/physical_infra_manager/refresh_parser.rb]: Ok. I Agree with you. Let It be the current name, then. :+1:

**Comment by agrare** [comment]
Thanks @douglasgabriel really awesome enhancement from schema all the way to the parser

**Pull request merged**